doc/man/pwquality.conf.5.pod

```diff
@@ -6,10 +6,18 @@ pwquality.conf - configuration for the libpwquality library
 
 =head1 SYNOPSIS
 
+Configuration files which can be adapted/changed by the user:
+
 F</etc/security/pwquality.conf>
 
 F</etc/security/pwquality.conf.d/*.conf>
 
+Default configuration files if there are no in /etc/security available:
```

Comment:
Please change: no -> none

And `F</etc/security>`

Reply:
done :-)